Redesigned UI refactor

5thAttemptCode opened the pull request:
#Modernized UI, improved UX, refactored code/DRYing it up

## Modernized UI
Refined the UI with a clean, modern aesthetic, utilizing a monochromatic color palette and accent colors to highlight key call-to-action buttons. Implemented a structured user flow guiding visitors through the site. The hero section features a bold headline to capture attention, followed by a section highlighting the community’s core values. The project section immediately after demonstrates how our work reflects these values. A concise purpose section leads into the About Us area, where users can navigate to a dedicated page to learn more about all contributors. The site concludes with a contact section, providing a clear endpoint for engagement. This flow is designed to guide users naturally from understanding our mission to exploring our projects and team.

###Hero section
<img width="1863" height="847" alt="Untitled" src="https://github.com/user-attachments/assets/f351e5e9-d475-4bcc-adf8-e090e3e8b381" />
Enhanced clarity and engagement with a bold headline, succinct description, and an accent-colored call-to-action that draws user attention.

###Core values
<img width="1678" height="663" alt="Untitled1" src="https://github.com/user-attachments/assets/c325179f-a806-4c5e-8c6a-06e88ab7be06" />
Presented core values through a simple and modern 3D block graphic

###Project section
<img width="1515" height="727" alt="Untitled2" src="https://github.com/user-attachments/assets/a3ed0133-ec01-4eea-b90b-1202f43a1f28" />
Standardized the background color of project images and replaced outdated visuals with modern, clean designs.

###Purpose
<img width="1471" height="677" alt="Untitled3" src="https://github.com/user-attachments/assets/6607c4e1-e8f8-407b-938f-c7118ba26135" />
Highlighting the community's purpose 

###About 
<img width="1580" height="715" alt="Untitled4" src="https://github.com/user-attachments/assets/49242b60-589e-4a2f-beba-ec28a72ff665" />
<img width="1483" height="745" alt="Untitled6" src="https://github.com/user-attachments/assets/ca28afaf-9b2b-4e16-be09-ac66694e7aa4" />
Replaced the previous timeline and media sections with a concise, clear About Us text, accompanied by a call-to-action button directing users to the Contributors page. The Contributors page has been modernized with a cleaner layout and subtle hover animations.

###Contact
<img width="1475" height="777" alt="Untitled5" src="https://github.com/user-attachments/assets/34da7f7e-a179-4c85-9f56-6bb786cf9a5f" />
Implemented a clean, straightforward contact form with clearly defined social media buttons.


##Improved UX

- Standardized all icons by adopting a consistent outlined style.
- Standardized contributor images by grayscaling them, reducing visual clutter and creating a cohesive look.
- Adopted a monochromatic color scheme to enhance clarity and keep users’ attention on the content.
- Highlighted the community’s values, work, and purpose to clearly communicate the website’s mission to users.
- Enhanced image titles and call-to-action labels to improve clarity for users.
- Refined the navigation for clarity, showing icons only where necessary on CTAs.


##DRYing up code

- Modularized components to improve reusability and maintainability.
- Organized data into arrays within utils for cleaner, more maintainable code.

5thAttemptCode committed:
github icon in nav bar was not centered when hovered

5thAttemptCode committed:
image slider for Contributors on mobile

5thAttemptCode committed:
adjusted image slider for mobile

5thAttemptCode committed:
made requested changes for pr

5thAttemptCode committed:
made requested changes for pr

5thAttemptCode committed:
new contributing member

5thAttemptCode committed:
please swipe-text only visible on mobile

5thAttemptCode committed:
WIP: UI redesign and refactor

5thAttemptCode committed:
refactor: merge redesigned UI into latest main

5thAttemptCode committed:
refactor: recovered new code

Green-Ranger11 commented on igned-ui-refactor:
Can we remove this please

Green-Ranger11 reviewed (changes requested):
Awesome stuff malo lava! Just a small change to remove a file accidentally committed and to fix the e2e please

5thAttemptCode committed:
test: update e2e tests to match new headings and contributors route

5thAttemptCode committed:
test: fix Hero heading text to match current UI

Green-Ranger11 committed:
chore: remove accidentally committed less help page dump

Green-Ranger11 committed:
fix: restore tailwind config as .js and replace broken skeleton variant classes

Rename tailwind.config.cjs back to .js since it uses ESM syntax.
Replace variant-filled-success/error/surface with Tailwind equivalents
since the skeleton tw-plugin breaks the build.

Green-Ranger11 commented:
Thanks! @5thAttemptCode
